### ISSUE TITLE
Consistency with the tutorial

### ISSUE THREAD

**Issue by YipingNUS** [FraLotito/continuous-bag-of-words#1]
Thanks for posting the code. A couple of questions/comments below:

1. In the tutorial, it seems the CBOW model has only 1 hidden layer, but in your code, you used 2.

> −logp(wi|C)=−logSoftmax(A(∑w∈Cqw)+b)

2. ln72, is it necessary to add the dimension to be summed? Otherwise it'll [sum the embeddings along all dimensions and get a scalar](https://stackoverflow.com/questions/44790670/torch-sum-a-tensor-along-an-axis)?

``embeds = sum(self.embeddings(inputs)).view(1,-1)``

**Comment by FraLotito** [comment]
Hi!

1) I use 2 linear layers and an embedding layer, if you want you can use only one linear layer (from embedding dimension to vocabulary dimension) and I think it would be perfectly fine.

2) In this case I used sum() which is different from torch.sum(). I think it would be better to use torch.sum() (in order to be consistent with pytorch functions) and pay attention to the dimension to be summed.

**Comment by YipingNUS** [comment]
Thanks for the reply :)
